src/go/rpk/pkg/cli/cmd/wasm/deploy_test.go

```diff
@@ -0,0 +1,214 @@
+					return map[string]sarama.TopicDetail{}, nil
+				},
+				createTopic: func(topic string, detail *sarama.TopicDetail, b bool) error {
+					_, exist := detail.ConfigEntries["cleanup.policy"]
+					require.True(t, exist)
+					require.Equal(t, kafka.CoprocessorTopic, topic)
+					return nil
+				},
+			},
+		}, {
+			name:	"it shouldn't create a coprocessor_internal_topic if it exist",
+			fileInfo: fileInfo{
+				name: "fileName.js",
+			},
+			pre: func(fs afero.Fs, fileInformation fileInfo) error {
+				return createMockFile(fs, fileInformation.name, fileInformation.content)
+			},
+			admin: &mockAdmin{
+				listTopics: func() (map[string]sarama.TopicDetail, error) {
+					result := make(map[string]sarama.TopicDetail)
+					result[kafka.CoprocessorTopic] = sarama.TopicDetail{}
+					return result, nil
+				},
+				createTopic: func(topic string, detail *sarama.TopicDetail, b bool) error {
+					require.NoError(t, fmt.Errorf("it shouldn't create topic"))
+					return nil
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			admin := func() (sarama.ClusterAdmin, error) {
+				if tt.admin != nil {
+					return tt.admin, nil
+				}
+				return &mockAdmin{
+					listTopics: func() (map[string]sarama.TopicDetail, error) {
+						result := make(map[string]sarama.TopicDetail)
+						result[kafka.CoprocessorTopic] = sarama.TopicDetail{}
+						return result, nil
+					},
+					createTopic: func(topic string, detail *sarama.TopicDetail, b bool) error {
+						return nil
+					},
+				}, nil
+			}
+			createProduce := func(_ bool, _ int32) (sarama.SyncProducer, error) {
+				produce := mocks.NewSyncProducer(t, nil)
+				if tt.failSendMessage == false {
+					produce.ExpectSendMessageWithCheckerFunctionAndSucceed(func(val []byte) error {
+						if tt.fileInfo.content != "" {
+							expectValue, _ := utils.WriteString([]byte{}, tt.fileInfo.content)
+							require.Equal(t, expectValue, val)
+						}
+						return nil
+					})
+				} else {
+					// it must fail 3 times for kafka retry policies.
+					produce.ExpectSendMessageAndFail(fmt.Errorf(""))
+					produce.ExpectSendMessageAndFail(fmt.Errorf(""))
+					produce.ExpectSendMessageAndFail(fmt.Errorf("kafka mock error"))
+				}
+				return produce, nil
+			}
+
+			fs := afero.NewMemMapFs()
+			if tt.pre != nil {
+				require.NoError(t, tt.pre(fs, tt.fileInfo))
+			}
+
+			var out bytes.Buffer
+			logrus.SetOutput(&out)
+			logrus.SetLevel(logrus.DebugLevel)
+			cmd := NewDeployCommand(fs, createProduce, admin)
+			cmd.SetArgs(append([]string{tt.fileInfo.name}, tt.args...))
+			err := cmd.Execute()
+			if tt.expectedErr != "" {
+				require.Contains(t, err.Error(), tt.expectedErr)
+				return
+			}
+			require.NoError(t, err)
+			for _, ln := range tt.expectedOutput {
+				require.Contains(t, out.String(), ln)
+			}
+		})
+	}
+}
+
+func createMockFile(fs afero.Fs, fileName string, fileContent string) error {
+	file, errFile := fs.Create(fileName)
```

Comment:
Rename `errFile` to just `err`. Always use `err`, there are only some situations where you'll need to name an `error` instance differently.

Reply:
got it